contrib/exporters/vpclogs/main.go

```diff
@@ -28,4 +28,5 @@ func main() {
 
 func init() {
 	core.TransformerHandlers.Register("vpclogs", sa.NewTransform, false)
+	core.AccRejHandlers.Register("aws", sa.NewAction, false)
```

Comment:
change to `action` and use the `Mangle` abstraction I introduced in #1994

Reply:
I don't see 1994.

Reply:
I assume you mean 1992. After you merge 1992 we can refactor. In the meantime, I'm leaving it independent of 1992 to work against origin/master.